sdk/python/kfp/v2/compiler/main.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2020 Google LLC
```

Comment:
Can this be added in a follow up PR?

Reply:
This is needed for the test which uses the cli `dsl-compile-v2`. Unless we want to split the test to a follow-up PR as well.

Reply:
Wouldn't a unit-test for the compiler to start with work?